evap/staff/templates/staff_semester_view.html

```diff
@@ -262,17 +262,21 @@ <h3>
                                     <th></th>
                                 </tr>
                             </thead>
-                        {% endif %}
-                        <tbody>
-                        {% for evaluation in evaluations %}
-                            <tr id="evaluation-row-{{ evaluation.id }}">
-                                {% include 'staff_semester_view_evaluation.html' with semester=semester evaluation=evaluation info_only=False %}
-                            </tr>
-                        {% empty %}
-                            <tr><td><i>{% trans 'There are no evaluations in this semester.' %}</i><td></tr>
-                        {% endfor %}
-                        </tbody>
-                    </table>
+                            <tbody>
+                            {% for evaluation in evaluations %}
+                                <tr id="evaluation-row-{{ evaluation.id }}">
+                                    {% include 'staff_semester_view_evaluation.html' with semester=semester evaluation=evaluation info_only=False %}
+                                </tr>
+                            {% endfor %}
+                            </tbody>
+                        </table>
+                    {% else %}
+                        <table>
+                            <tbody>
+                                <tr><td><i>{% trans 'There are no evaluations in this semester.' %}</i><td></tr>
```

Comment:
```suggestion
                                <tr><td><i>{% trans 'There are no evaluations in this semester.' %}</i></td></tr>
```
not your fault, but as you're on it anyway...